cms/djangoapps/export_course_metadata/storage.py

```diff
@@ -17,4 +17,7 @@ def __init__(self):
         bucket = settings.COURSE_METADATA_EXPORT_BUCKET
         super().__init__(bucket_name=bucket, custom_domain=None, querystring_auth=True)
 
-course_metadata_export_storage = get_storage_class(settings.COURSE_METADATA_EXPORT_STORAGE)()
+course_metadata_export_storage = resolve_storage_backend(
+    storage_key="course_metadata_export_storage",
```

Comment:
```suggestion
    storage_key="course_metadata_export",
```
The storage suffix is redundant given that this setting will be in the `STORAGES` top level letting.

Reply:
Done ✅